Moodis/Moodis/Feature/Login/LoginForm.cs

```diff
@@ -7,99 +9,58 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
-using moodis;
-using System.Text.RegularExpressions;
 
 namespace Moodis.Feature.Login
 {
     public partial class LoginForm : Form
     {
-
+        LoginViewModel loginViewModel;
         public const String usernameEmpty = "Username field is empty!";
         public const String passwordEmpty = "Password field is empty!";
         public const String userNotFound = "User not found!";
-        public const String passwordsNotSame = "Passwords must be the same!";
-        public const String created = " was created!";
-        public const String exists = " alreadyExists!";
-        public const String strongerPassword = "Password must be stronger!";
-
-        private readonly LoginViewModel loginViewModel;
         public LoginForm()
         {
             InitializeComponent();
             loginViewModel = new LoginViewModel();
+            loginViewModel.AddUser("admin", "admin");
         }
 
-        private void LoginButton_Click(object sender, EventArgs e)
+        private void ButtonSignIn_Click(object sender, EventArgs e)
         {
-            if (string.IsNullOrWhiteSpace(usernameField.Text))
+            if(string.IsNullOrEmpty(textBoxUsername.Text))
```

Comment:
Why change from IsNullOrWhiteSpace to IsNullOrEmpty ? IsNullOrWhiteSpace method covers IsNullOrEmpty also it checks whether there are space/tab symbols entered.

Reply:
My bad, selected the wrong statement